ModelViewIntentSample/search/src/main/java/com/novoda/movies/mvi/search/data/SearchBackend.kt

```diff
@@ -3,12 +3,16 @@ package com.novoda.movies.mvi.search.data
 import com.novoda.movies.mvi.search.domain.SearchResults
 import io.reactivex.Single
 
+interface MovieDataSource {
```

Comment:
What's the motivation for declaring an interface here?

Reply:
It was to avoid depending on an interface for our tests. 
Apparently with mockito we can also mock a class, so I can revert this